tensorboard/compat/proto/proto_test.py

```diff
@@ -185,7 +207,7 @@ def test_each_proto_matches_tensorflow(self):
             self.assertEquals(
                 diff,
                 "",
-                "{} and {} did not match:\n{}".format(tf_path, tb_path, diff),
+                MATCH_FAIL_MESSAGE_TEMPLATE.format(tf_path, tb_path, diff),
             )
```

Comment:
As written, this prints the diff twice (output below). It looks like the
intent of this format string was to display the “NOTE!” below the diff
because the note is more actionable. That makes sense to me. In that
case, can we change the call site to just

```python
            if diff:
                self.fail(MATCH_FAIL_MESSAGE_TEMPLATE.format(diff))
```

?

<details>
<summary>Current test failure output</summary>

```
FAIL: test_each_proto_matches_tensorflow (__main__.ProtoMatchTest)
test_each_proto_matches_tensorflow (__main__.ProtoMatchTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/RUNFILES_PATH/tensorboard/compat/proto/proto_test.py", line 210, in test_each_proto_matches_tensorflow
    MATCH_FAIL_MESSAGE_TEMPLATE.format(tf_path, tb_path, diff),
AssertionError: 
tensorflow.core.framework.allocation_description_pb2 and tensorboard.compat.proto.allocation_description_pb2 did not match: --- 
+++ 
@@ -43,7 +43,7 @@
   java_package: "org.tensorflow.framework"
   java_outer_classname: "AllocationDescriptionProtos"
   java_multiple_files: true
-  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework"
+  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/allocation_description_go_proto"
   cc_enable_arenas: true
 }
 syntax: "proto3"


NOTE!
====
This is expected to happen when TensorFlow updates their proto definitions.
We pin copies of the protos, but TensorFlow can freely update them at any
time.

The proper fix is:

1. In your TensorFlow clone, check out the version of TensorFlow whose
   protos you want to update (e.g., `git cehckout v2.2.0-rc0`)
2. In your tensorboard repo, run:

    ./tensorboard/compat/proto/update.sh PATH_TO_TENSORFLOW_REPO

3. Review and commit any changes.
:
- --- 
- +++ 
- @@ -43,7 +43,7 @@
-    java_package: "org.tensorflow.framework"
-    java_outer_classname: "AllocationDescriptionProtos"
-    java_multiple_files: true
- -  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework"
- +  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/allocation_description_go_proto"
-    cc_enable_arenas: true
-  }
-  syntax: "proto3"


----------------------------------------------------------------------
Ran 2 tests in 0.003s
```

</details>